src/e2e-test/java/io/cdap/plugin/gcsmove/stepsdesign/GCSMove.java

```diff
@@ -37,6 +40,13 @@
  */
 public class GCSMove implements E2EHelper {
 
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='pipeline-configure-modeless-btn']")
+  public static WebElement pipelineConfigureButton;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='tab-head-Pipeline alert']")
+  public static WebElement pipelineAlertsItem;
```

Comment:
Why are we adding these here instead of the locators file?

Reply:
Added another class for GCSDoneFileMarger related locators.